boards/avr-rss2/Makefile.dep

```diff
@@ -0,0 +1,6 @@
+USEMODULE += boards_common_atmega
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += bmx280
+  USEMODULE += saul_gpio
```

Comment:
What happened to `USEMODULE += ds18`?

Reply:
Oh a relevant question. I did remove it as as the Dalllas DS18B20 is replaced by the BME280 on all new boards. The tests/saul was complaining and was creating confusion.

Need some Travis help/review. Will use --fixup in coming work.

When PR is over yes the EUI64 stuff needs to be revisited for EUI HW chips.

Reply:
Travis is just complaining about the presence of *Fixup* commits, this is fine.
But they have to be squashed before the merge.

Ok, so my module doesn't have the BME280 then. I'll have to rely on your testing then.
Is that the I2C version? Maybe you can give https://github.com/RIOT-OS/RIOT/pull/8383 a try :wink: 

The EUI64 stuff can be added by a later PR once the EEPROM/EUI-API is finalized.

Reply:
To be clear. Your have a bme280 on the 2.4 board. It sits on i2c and should work.
The 8383 PR seems to deal with SPI.

Reply:
Yes. Please tell if it's time to squash. So we after that can proceed with the EUI stuff.